src/main/java/com/igot/cb/designation/service/DesignationService.java

```diff
@@ -1,8 +1,15 @@
 package com.igot.cb.designation.service;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.igot.cb.pores.dto.CustomResponse;
+import com.igot.cb.pores.util.ApiResponse;
 import org.springframework.web.multipart.MultipartFile;
 
 public interface DesignationService {
 
- public void loadDesignationFromExcel(MultipartFile file);
+  public void loadDesignationFromExcel(MultipartFile file);
```

Comment:
We suppose to use csv ?

Reply:
yeah we are uploading the csv now, implemented the csv upload in the method